default/python/universe_generation/natives.py

```diff
@@ -95,6 +95,7 @@ def generate_natives(native_freq, systems, empire_home_systems):
             # if no, and there is at least one available focus, just take the first of the list
             # otherwise don't set any focus
             fo.planet_set_focus(candidate, available_foci[0])
+            print("preferred focus '",preferred_focus,"' not available, focus on '",available_foci[0],"' instead")
```

Comment:
anyway, yeah, focus not available .. not sure why though
<img width="1104" alt="focus not available log" src="https://user-images.githubusercontent.com/37298400/149681462-540e5072-fe44-40cd-a9ca-21533ffee227.png">